diff --git motorApp/MotorSrc/motordevCom.cc motorApp/MotorSrc/motordevCom.cc
index 9650d0c1..3f73a432 100644
--- motorApp/MotorSrc/motordevCom.cc
+++ motorApp/MotorSrc/motordevCom.cc
@@ -463,8 +463,8 @@ epicsShareFunc long motor_start_trans_com(struct motorRecord *mr, struct board_s
     motor_call->type = UNDEFINED;
     motor_call->mrecord = (struct dbCommon *) mr;
     motor_call->message[0] = (char) NULL;
-    motor_call->postmsgptr = (char) NULL;
-    motor_call->termstring = (char) NULL;
+    motor_call->postmsgptr = (char*) NULL;
+    motor_call->termstring = (char*) NULL;
     
     return (0);
 }
